assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview/index.js

```diff
@@ -95,6 +96,11 @@ const Overview = ( {
 		select( CORE_MODULES ).isModuleActive( 'analytics' )
 	);
 
+	const isAnalyticsGatheringData = useInViewSelect( ( select ) =>
+		analyticsModuleConnected
+			? select( MODULES_ANALYTICS ).isGatheringData()
+			: false
```

Comment:
It's a minor point, but this could be marginally simplified by using logical AND here instead:

```suggestion
		analyticsModuleConnected && select( MODULES_ANALYTICS ).isGatheringData()
```

Reply:
Thanks, updated 👍